ggu/main.c

```diff
@@ -746,16 +755,23 @@ ggu_stage2(void *arg)
 	int ret;
 
 	/*
-	 * Setup the ntuple filters that assign the GK-GT packets
+	 * Setup the ntuple filters that assign the GT-GK packets
 	 * to its queue for both IPv4 and IPv6 addresses.
 	 */
 	if (hw_filter_ntuple_available(&ggu_conf->net->back)) {
-		return ntuple_filter_add(&ggu_conf->net->back,
+		ret = ntuple_filter_add(&ggu_conf->net->back,
 			ggu_conf->net->back.ip4_addr.s_addr,
 			ggu_conf->ggu_src_port, UINT16_MAX,
 			ggu_conf->ggu_dst_port, UINT16_MAX,
 			IPPROTO_UDP, ggu_conf->rx_queue_back,
 			ipv4_configured, ipv6_configured);
+		/*
+		 * XXX Returning here without adding an ACL
+		 * rule for IPv6 is a known bug that is fixed
+		 * in the next commit.
```

Comment:
The label `TODO` is for things that need to be removed for release, whereas `XXX` is for things that we are aware, but will not solve now. So we should use `TODO` label here.

If this comment upsets you for me being too picky, ignore it. I'll count it on the house. :)